dashboard/src/components/jobs/parameters/TimeParameters.vue

```diff
@@ -12,6 +12,9 @@
       >
         <option v-for="tz in timezoneList" :key="tz">{{ tz }}</option>
       </select>
+      <help
+        helpText="The timezone used in your data. If your data does not account for daylight savings, use the appropriate fixed-offset. (e.g. Etc/GMT-7)"
```

Comment:
```suggestion
        helpText="The timezone used in your data. If your data does not account for daylight savings, use the appropriate fixed-offset. (e.g. Etc/GMT+7)"
```

I don't know the exact reasoning but +7 is the right zone for Arizona